addons/de/xrechnung/instructions_test.go

```diff
@@ -0,0 +1,100 @@
+package xrechnung_test
+
+import (
+	"testing"
+
+	"github.com/invopop/gobl/addons/de/xrechnung"
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/pay"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPaymentInstructions(t *testing.T) {
+	t.Run("valid SEPA credit transfer", func(t *testing.T) {
+		inv := &bill.Invoice{
+			Payment: &bill.Payment{
+				Instructions: &pay.Instructions{
+					Key: xrechnung.KeyPaymentMeansSEPACreditTransfer,
+					CreditTransfer: []*pay.CreditTransfer{
+						{
+							IBAN: "DE89370400440532013000",
+							BIC:  "DEUTDEFF",
+						},
+					},
+				},
+			},
+		}
+		assert.NoError(t, xrechnung.ValidatePaymentInstructions(inv))
```

Comment:
I'd run the validation on the whole Invoice to make sure nothin else is broken. Please see the other examples.